### PR TITLE
Added module for direct download of csv and edited run.py

### PR DESCRIPTION
**Pull request by charlottekostelic**
Added module to download csv from Google Sheets (see #4). Edited run.py so user can pass next control # in sequence as an argument. Edited README.md to reflect changes to run.py (see #3).

### CODE REVIEW

File: bagel/download.py
Comment (klinga) [inline bagel/download.py:1]: have the`converters` arg in `.read_csv` fixed the new line problem?

File: README.md [hunk 1]
```diff
@@ -1,12 +1,25 @@
 # Bagel
  Scripts for ingesting data & producing MARC21 records for BPL Board Game Library
 
+## Submission data
+
+ Submission data is collected in [this spreadsheet](https://docs.google.com/spreadsheets/d/1Z8kWlHZXbnzP7OQWK3nLy7DEdxedPv2pW2pYodh6_dY/edit#gid=1152172600).
+
+## Setup
+ This project requires Python 3.12.
```
[the remark this answers is not in the document]
Comment (klinga) [reply README.md:9]: Oh, I see now that pandas/numpy was only now added. No issues with `distutils` then 😉

**Review by klinga — COMMENTED** [review]
Good work!
I only have few suggestions for the README instruction. Please see the details, make changes if needed, commit, push to your branch. Will be reflected in this PR.

File: bagel/download.py
Comment (charlottekostelic) [reply bagel/download.py:1]: Yes, adding the `converters` arg in `.read_csv` fixed the new line issue but it only accounts for new lines in the Description/Summary field. If we anticipate that BPL staff might add new lines in other fields I can change it to apply to other columns.

Comment (klinga) [reply bagel/download.py:1]: The Description/Summary is the most likely element for this to occur. If this changed can parse correctly the entire sheet with hundreds of rows, we should be good. Thank you!

**Pull request merged**